### ISSUE TITLE
Importing SQLAlchemy-Utils with SQLAlchemy 1.3.4 emits a warning

### ISSUE THREAD

**Issue by m-aciek** [kvesteri/sqlalchemy-utils#374]
```python
>>> import sqlalchemy
>>> import sqlalchemy_utils
/.../site-packages/sqlalchemy/sql/functions.py:68: SAWarning: The GenericFunction 'array_agg' is already registered and is going to be overriden.
  "is going to be overriden.".format(identifier))
```

Is it possible for SQLAlchemy-Utils to register the function at some other name or register it without the warning?

**Comment by Viste** [comment]
Confrim!

**Comment by Veles** [comment]
Confrim!

**Comment by m-aciek** [comment]
https://github.com/kvesteri/sqlalchemy-utils/blob/master/sqlalchemy_utils/expressions.py#L115 This is the line which causes warning.

`array_agg` is not documented in SQLAlchemy-Utils docs. It may be a fix of thing, that is already fixed in SQLAlchemy core, https://github.com/sqlalchemy/sqlalchemy/issues/4107? @kvesteri, could you confirm?

**Comment by m-aciek** [comment]
For the record, this commit introduced `array_agg` in SQLAlchemy-Utils: https://github.com/kvesteri/sqlalchemy-utils/commit/89852542f212571714bfbb8acec55cdc99621f84.

**Comment by VizualAbstract** [comment]
Confirm...

**Comment by aaron-suarez** [comment]
+1 for this problem

**Comment by m-aciek** [comment]
Closing as fix is released in 0.34.0. Thank you @kvesteri.
